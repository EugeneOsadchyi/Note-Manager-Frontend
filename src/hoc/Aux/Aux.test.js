import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Aux from './Aux';

describe('Aux', () => {
  const aux = shallow((
    <Aux>
      <h1>
        Header 1
      </h1>
      <h2>
        Header 2
      </h2>
      <h3>
        Header 3
      </h3>
    </Aux>
  ));

  it('renders properly', () => {
    expect(shallowToJson(aux)).toMatchSnapshot();
  });
});
