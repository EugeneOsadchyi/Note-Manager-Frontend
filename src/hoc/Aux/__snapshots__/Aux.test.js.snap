// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aux renders properly 1`] = `
Array [
  <h1 />,
  <h2 />,
  <h3 />,
]
`;
